src/PiCraft.jl

```diff
@@ -39,10 +40,12 @@ module world
     end
 
     function setBlock(x,y,z,block::PiCraft.Block, data)
+        (x, y, z) = (floor(Int64, x), floor(Int64, y), floor(In64, z))
         PiCraft.mc_send("world.setBlock($x,$y,$z,$(block.id),$(block.data),$data)", false)
     end
 
```

Comment:
`setBlock` needs to accept any value but since block position is an Int vector must be floored.